include/swift/Frontend/Frontend.h

```diff
@@ -433,6 +449,57 @@ class CompilerInstance {
   /// Frees up the ASTContext and SILModule objects that this instance is
   /// holding on.
   void freeContextAndSIL();
+
+private:
+  /// Load stdlib & true if should continue, i.e. no error
```

Comment:
I think you [accidentally](http://knowyourmeme.com/memes/i-accidentally) a verb. :-)

Reply:
Yes! You a great catch.